package.json

```diff
@@ -17,7 +17,9 @@
     "postbuild": "ts-node scripts/postBuild.ts",
     "prettier": "prettier --write --config prettier.config.js .",
     "prettier:check": "prettier --config prettier.config.js --check .",
-    "prettier:scripts": "prettier --write ./scripts"
+    "prettier:scripts": "prettier --write ./scripts",
+    "lint:js": "eslint . --ext .js,.tsx,.ts",
+    "lint:js:fix": "npm run lint --fix"
```

Comment:
I think it should be `lint:js` not `lint`

Reply:
Fix in 0243ea32fcf647c05662d9fd47117bf66d871c21